sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AzureMonitorSpanExporterProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.azure.monitor.opentelemetry.exporter.implementation;
+
+import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+import java.util.Collection;
+
+public class AzureMonitorSpanExporterProvider implements ConfigurableSpanExporterProvider {
+    @Override
+    public SpanExporter createExporter(ConfigProperties configProperties) {
+        if (configProperties.getBoolean(AzureMonitorExporterProviderKeys.INTERNAL_USING_BUILDER, false)) {
+            return MarkerSpanExporter.INSTANCE;
+        }
+        return new AzureMonitorExporterBuilder().buildTraceExporter();
+    }
+
+    @Override
+    public String getName() {
+        return AzureMonitorExporterProviderKeys.EXPORTER_NAME;
+    }
+
+    public enum MarkerSpanExporter implements SpanExporter {
```

Comment:
Can we rename it to InternalSpanExporter?

Reply:
same applies to other exporter type.

Reply:
I sort of like the name "Marker" because it just marks the place where we want to put the real exporter once we have enough info to instantiate it